README.md

```diff
@@ -2,9 +2,12 @@
 
 ## APIs
 
+- [Contributions](/contributions)
+- [Crypto Site](/crypto-site)
```

Comment:
Can we just call it `Crypto`? You'll have to change the folder name and also the link to the folder -- what you see inside () brackets here